reddit_gold/controllers.py

```diff
@@ -79,12 +70,6 @@ class GoldApiController(RedditController):
                    VGold(),
                    VModhash(),
                    deal=VLength('deal', 100))
-    def POST_claim_gold_partner_deal_code(self, responder, deal):
```

Comment:
Ahh, makes sense.  I originally did some overzealous deletion and removed the entire class.